examples/crm/src/settings/SettingsPage.tsx

```diff
@@ -80,13 +81,46 @@ const SettingsForm = ({
     isEditMode: boolean;
     setEditMode: (value: boolean) => void;
 }) => {
+    const [update] = useUpdate();
+    const notify = useNotify();
+    const { identity, refetch } = useGetIdentity();
     const { isDirty } = useFormState();
     const [openPasswordChange, setOpenPasswordChange] = useState(false);
 
+    if (!identity) return null;
+
     const handleClickOpenPasswordChange = () => {
         setOpenPasswordChange(true);
     };
 
+    const handleAvatarUpdate = async (values: any) => {
+        await update(
+            'sales',
+            {
+                id: identity.id,
+                data: values,
+                previousData: identity,
+            },
+            {
+                onSuccess: data => {
+                    // Update local user
+                    localStorage.setItem(
```

Comment:
**suggestion:** The local user update is (almost) the same has `SettingsPage#handleOnSubmit`, can you factorize them

Reply:
I think it's overkill, I'll see if it's used somewhere else.